src/google_heatmap.js

```diff
@@ -1,8 +1,9 @@
-import fp from 'lodash/fp';
+import map from 'lodash.map';
+import reduce from 'lodash.reduce';
```

Comment:
Instead of importing functions map and reduce from lodash, we can simply call native map reduce functions. If you agree i can change in this PR

Reply:
@ZAKdev If you can make a separate PR doing that, I can handle the rest. This is my first option, but the second one would be using all _our own utils_ instead. So for both options the change you're proposing would be great.

Reply:
@ZAKdev Either way I go, your change independently will decrease the size of the build.

Reply:
@itsmichaeldiego, Ok i will do it in separate PR, i will open a PR soon and let me know if i can help you more to decrease the size of build.

Reply:
@ZAKdev Thanks, it will be really helpful for you to change the reduce and map given that you can test the changes in the heatmap and see if it works, thanks for that! I will be waiting on this, do you think you can do it today? ♥️